Client/Public/JS/Controllers/GameController.js

```diff
@@ -1220,13 +1259,20 @@ function updateWithDataFromServer(data) {
 // Initializes the autosave timer to ensure user data persistence.
 //                  User data is autosaved every 15 seconds.
 function autosaveTimer() {
+    if (is_host) {
```

Comment:
NTS-3: Since we already do the check for if a user is host in the save function, we can remove this check.